zscript/StupidAchievements.zs

```diff
@@ -507,15 +532,23 @@ class sa_NoAnimationTask : sa_Task
     int borderX = x;
     int borderY = y;
 
+	int iconWidth = boxheight;
+	int iconHeight = iconWidth;
+	int iconX =  boxX - iconWidth * 1.1 ;
+	int iconY =  textY - mAchievement.margin / 2;
```

Comment:
Icon is put to the left of the text box, it won't be displayed if achievements are displayed on the left side of the screen.
In this case, icons should be either displayed to the right of the text box, or text box should be moved to the right.

Reply:
I know that the icon cannot be displayed in notification. Actually I wish to display it like Steam, GOG and other platforms, but I'm worried about it hides status bar and disturbs gameplay.